cmd/snap/cmd_assert.go

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/i18n"
+	"github.com/ubuntu-core/snappy/logger"
+)
+
+type assertOptions struct {
+	AssertionFile string `positional-arg-name:"assertion file" description:"assertion file"`
+}
+
+type cmdAssert struct {
+	assertOptions `positional-args:"true" required:"true"`
+}
+
+var (
+	shortAssertHelp = i18n.G("Assert tries to add an assertion to the system.")
+	longAssertHelp  = i18n.G("This command tries to add an assertion to the system assertion database.")
```

Comment:
@elopio it's like event database or rule database, using a plural there is odd english wise afaik